smarts/core/agent.py

```diff
@@ -58,6 +58,15 @@ def act(self, obs):
 
         raise NotImplementedError
 
+    def step(self, state, action, reward, next_state, done, info):
+        return {}
+
+    def save(self, path):
```

Comment:
As above, for this and `load()`, the type of the parameter (presumably `str`) and return value (if any) should be explicitly indicated.